zeebe/exporters/operate-exporter/pom.xml

```diff
@@ -84,6 +89,11 @@
       <artifactId>operate-common</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>operate-importer-common</artifactId>
```

Comment:
Similar question about this dependency. Why do we need it? I think it's OK to keep it for now, but would be nice to create an issue to clean this up later.